app/src/main/java/org/simple/clinic/bp/assignbppassport/BpPassportEffect.kt

```diff
@@ -0,0 +1,5 @@
+package org.simple.clinic.bp.assignbppassport
+
+sealed class BpPassportEffect
+
+data class OpenPatientEntryScreen(val bpPassportNumber: Int) : BpPassportEffect()
```

Comment:
The BP passport number we use is not an int. It's an `Identifier`, can you take a look at the existing register new patient functionality from the search. Looks like we don't even have to use activity results to open the screen, but instead use the `OngoingLoginEntry`. So, please take a look at the `PatientSearchResultsScreen` Mobius implementation to see how the new patient registration works.